src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -397,16 +431,131 @@ void EIRPlantLoopHeatPump::doPhysics(EnergyPlusData &state, Real64 currentLoad)
         }
     }
 
+    // check the curve values, reset to zero if negative
+    this->capModFTCurveCheck(state, loadSideOutletSetpointTemp, capacityModifierFuncTemp);
+}
```

Comment:
Created a capacity modifier curve value check function and used in `calcAvailableCapacity()` only.